### PR TITLE
Fixing tag code

### LINKED ISSUE #270: Add github workflow to publish lambda images to GHCR
### Description
APHL has asked us to make dummy instances of the TTC and augmentation lambdas available to them by 3/3 so that they can test out integrating the lambdas into their workflows. We need to update our build + publish [workflow](https://github.com/CDCgov/dibbs-text-to-code/blob/main/.github/workflows/docker-build-ghcr.yml) to include both TTC and augmentation lambdas. 

### Notes
It may be worth looking at what Refiner has done for their workflow, [here](https://github.com/CDCgov/dibbs-ecr-refiner/blob/main/.github/workflows/docker-image-push.yml), to see if we can reuse it. Jake would also be a good resource if you have additional questions.

For now, APHL just needs dummy versions of the lambdas for testing purposes so feel free to add them to the repo.

### PR DESCRIPTION
**Pull request by robertandremitchell**
## Description

Fixes error with tags not generating. 

## Related Issues

Closes #270 

## Additional Notes

[Add any additional context or notes that reviewers should know about.]

<--------------------- REMOVE THE LINES BELOW BEFORE MERGING --------------------->

## Checklist

Please review and complete the following checklist before submitting your pull request:

- [ ] I have ensured that the pull request is of a manageable size, allowing it to be reviewed within a single session.
- [ ] I have reviewed my changes to ensure they are clear, concise, and well-documented.
- [ ] I have updated the documentation, if applicable.
- [ ] I have added or updated test cases to cover my changes, if applicable.
- [ ] I have minimized the number of reviewers to include only those essential for the review.

## Checklist for Reviewers

Please review and complete the following checklist during the review process:

- [ ] The code follows best practices and conventions.
- [ ] The changes implement the desired functionality or fix the reported issue.
- [ ] The tests cover the new changes and pass successfully.
- [ ] Any potential edge cases or error scenarios have been considered.

### CODE REVIEW

File: .github/workflows/docker-image-push.yml [hunk 1]
```diff
@@ -73,7 +73,11 @@ jobs:
         id: vars
         run: |
           # Get the branch/ref and version tag
-          REF_NAME="${{ github.event.inputs.ref || github.ref_name }}"
+          REF_NAME="${{ github.ref_name }}"
+          if [ "${{ github.event_name }}" = "workflow_dispatch" ] && [ -n "${{ github.event.inputs.ref }}" ]; then
+            REF_NAME="${{ github.event.inputs.ref }}"
+          fi
```
Comment (m-goggins) [inline .github/workflows/docker-image-push.yml]: I'm surprised that the previous code didn't work. Could it be because it was not set as an `env`? 

```
env:
  REF_NAME: ${{ github.event.inputs.ref || github.ref_name }}
  VERSION_TAG: ${{ github.event.inputs.version_tag }}

run: |
  if [ -z "$VERSION_TAG" ]; then
    TAG=$(echo "$REF_NAME" \
          | tr '[:upper:]' '[:lower:]' \
          | tr '/' '-' \
          | tr -cd 'a-z0-9._-')
  else
    TAG="$VERSION_TAG"
  fi

  echo "version_tag=$TAG" >> "$GITHUB_OUTPUT"
```

Comment (robertandremitchell) [reply .github/workflows/docker-image-push.yml]: makes sense, definitely happy to try that first. I can't really see why it wouldn't work either
